apiary.apib

```diff
@@ -1403,6 +1403,13 @@ A workflow has the following attributes
 *id*, *created_at*, *updated_at*, *workflow_version*, *content_language*,
 and *classifications_count* are assigned by the API
 
+Three parameters: _grouped_, _prioritized_, and _pairwise_ configure
```

Comment:
I added a not about the default. I think it'd be reasonable for people to assume none of the optionally exclude the others.